sdk/eventgrid/eventgrid/README.md

```diff
@@ -1,119 +1,260 @@
-## Azure EventGridClient SDK for JavaScript
+# Azure Event Grid client library for JavaScript
 
-This package contains an isomorphic SDK for EventGridClient.
+[Azure Event Grid](https://azure.microsoft.com/en-us/services/event-grid/) is a cloud-based service that provides reliable event delivery at massive scale.
+
+Use the client library to:
+
+- Send events to Event Grid using either the Event Grid, Cloud Events 1.0 schemas, or a custom schema
+- Decode and process events which were delivered to an Event Grid handler
+- Generate Shared Access Signatures for Event Grid topics
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventgrid/eventgrid/) |
+[Package (NPM)](https://www.npmjs.com/package/@azure/eventgrid) |
+[API reference documentation](https://aka.ms/azsdk-js-eventgrid-ref-docs) |
+[Product documentation](https://docs.microsoft.com/en-us/azure/event-grid/) |
+[Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventgrid/eventgrid/samples)
```

Comment:
@ellismg this directory does not exist and causing link verification in CI to fail.

Reply:
> @ellismg this directory does not exist and causing link verification in CI to fail.

Huh!  Indeed - I'm not sure where the samples are (I don't see them locally either, perhaps I missed `git add`'ing them at some point and they got blown away and I didn't notice.

Let me rewrite them and re-submit.

For my own edification, should this have been caught during the PR?  All the checks had passed but I'm not sure if there's something else I should have been looking at.

Reply:
@ellismg Thanks! I think it should have been caught in the PR pipeline, so it is weird that it was all green. cc @praveenkuttappan @KarishmaGhiya.